packages/src/streamdeck/index.ts

```diff
@@ -14,6 +14,9 @@ export function pkg<TServer>(ws: WsProvider<TServer>) {
     SettingsUI: () => import("./Settings"),
   });
 
+  if (localStorage.getItem("SDWS_PORT") !== null)
+    ctx.startServer(Number(localStorage.getItem("SDWS_PORT")));
+
```

Comment:
this should be done in `createCtx` and `SDWS_PORT` should be in a constant

Reply:
donezo